docs/topics/fake-app-spec.rst

```diff
@@ -0,0 +1,34 @@
+.. _fake-app-spec:
+
+===============
+ Fake App Data
+===============
+
+The `generate_apps_from_spec` command-line tool loads a JSON file containing an
+array of fake app objects. The fields that can be specified in these objects
+are:
+
+type
+    One of "hosted", "web", or "privileged", to specify a hosted app,
+    unprivileged packaged app, or privileged packaged app.
+
+status
+    An integer representing the app status, as listed in
+    ``mkt.constants.base``.
```

Comment:
To make this easier to read this could be the api status name and converted during processing. Make it easier for the humans.